manifests/cluster-network-addons/0.27.7/network-addons-config-example.cr.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: networkaddonsoperator.network.kubevirt.io/v1alpha1
+kind: NetworkAddonsConfig
+metadata:
+  name: cluster
+spec:
+  imagePullPolicy: IfNotPresent
+  kubeMacPool: {}
```

Comment:
aint there was mac v tap around ? or on other version / i am wrong

Reply:
This is version 0.27.7.  The was not macvtap back then.